gems/pending/appliance_console/database_replication_standby.rb

```diff
@@ -0,0 +1,82 @@
+require 'appliance_console/logging'
+require 'appliance_console/prompts'
+require 'appliance_console/database_configuration'
+require 'appliance_console/database_replication'
+
+module ApplianceConsole
+  class DatabaseReplicationStandby < DatabaseReplication
+    include ApplianceConsole::Logging
+
+    REGISTER_CMD = 'repmgr standby register'.freeze
+
+    attr_accessor :appliance_pg_data, :standby_host
```

Comment:
👍 Doing this will resolve many of the issues regarding host naming you've raised.